File: dotnet/MicrosoftGraph/Models/Importance.cs
using System.Runtime.Serialization;
using System;
namespace GitHubTodoDemo.MicrosoftGraph.Models {
    public enum Importance {
        [EnumMember(Value = "low")]
        Low,
        [EnumMember(Value = "normal")]
        Normal,
        [EnumMember(Value = "high")]
        High,
    }
}

File: php/MicrosoftGraph/Models/Importance.php
<?php

namespace Baywet\Githubtododemo\Microsoftgraph\Models;

use Microsoft\Kiota\Abstractions\Enum;

class Importance extends Enum {
    public const LOW = 'low';
    public const NORMAL = 'normal';
    public const HIGH = 'high';
}
